src/tools/cephfs/cephfs-shell

```diff
@@ -615,6 +614,13 @@ class CephFSShell(Cmd):
         """
         Copy a local file/directory to CephFS.
         """
+        if not os.path.isfile(args.local_path):
+            if not os.path.isdir(args.local_path):
+                set_exit_code_msg(errno.ENOENT,
+                                  msg=f"error: "
+                                      f"{args.local_path.decode('utf-8')}: "
+                                      f"No such file or directory")
+                return
         root_src_dir = args.local_path
         root_dst_dir = args.remote_path
```

Comment:
Add an extra newline after `return` to separate the code block?

```suggestion

                 return

        root_src_dir = args.local_path
        root_dst_dir = args.remote_path
```

Reply:
Okay

Reply:
I've made the suggested change, thanks!